experiment/simulation/js/add_numbers.js

```diff
@@ -0,0 +1,339 @@
+const languages = ['py', 'js', 'c'];
+
+const nextButton = document.getElementById('next');
+const prevButton = document.getElementById('prev');
+const runButton = document.getElementById('run');
+const resetButton = document.getElementById('reset');
+
+const stackElement = document.getElementById('container-stack');
+const stackText = document.getElementById('stack-depth');
+const resultElement = document.getElementById('result');
+
+let basecaseOperatorSelect = document.getElementById(`basecase-operator-${languages[0]}`);
+let basecaseSelect = document.getElementById(`basecase-${languages[0]}`);
+let basecaseReturnSelect = document.getElementById(`basecase-return-${languages[0]}`);
+let recursecaseSelect = document.getElementById(`recursecase-${languages[0]}`);
+let variableSelect = document.getElementById(`variable-${languages[0]}`);
+let operatorSelect = document.getElementById(`operator-${languages[0]}`);
+let nSelect = document.getElementById(`n-${languages[0]}`);
+
+let stackHTML = '';
+let resultContent = '';
+let stepstoexecute = -1;
+let executedsteps = 0;
+
+let maxStackDepth = parseInt(nSelect.value) + 1;
+stackText.textContent = maxStackDepth;
+
+let basecaseOperator = changeOperator(basecaseOperatorSelect.value);
+let basecase = parseInt(basecaseSelect.value);
+let basecaseReturn = parseInt(basecaseReturnSelect.value);
+let recursecase = recursecaseSelect.value;
+let variable = variableSelect.value;
+let operator = operatorSelect.value;
+
+languages.forEach(language => {
```

Comment:
Do we need separate loops and separate change handlers to update the value of each attribute or all of them can be combined into a single change handler function?